app/js/components/LabResultEntry.jsx

```diff
@@ -80,9 +81,25 @@ export class LabResultEntry extends PureComponent {
   renderForm(selectedLabConcept) {
     const { patientHeaderDetail } = this.state;
     const {
-      CONSTANTS, conceptMembers, history: { location: { state } },
+      CONSTANTS, conceptMembers, patients, history: { location: { state } },
     } = this.props;
 
+    const selectedPatient = patients[patientHeaderDetail.uuid] || {};
```

Comment:
Please can you wrap all this in a function, also maybe add a comment on the function that says what's it's doing or, give the function a very descriptive name. I don't know what all that code does.